app/lib/actions/katello/actions_common_jail.rb

```diff
@@ -0,0 +1,62 @@
+module Actions
+  module Katello
+    module ActionsCommonJail
+      def content_view_id
+        input['content_view']['id']
+      end
+
+      def content_view_name
+        input['content_view']['name']
+      end
+
+      def content_view_label
+        input['content_view']['label']
+      end
+
+      def organization_id
+        input['organization']['id']
+      end
+
+      def organization_name
+        input['organization']['name']
+      end
+
+      def organization_label
+        input['organization']['label']
+      end
+
+      def self.included(base)
+        super
+        base.instance_eval do
+          apipie :class, "A class representing #{base} object" do
+            desc 'This object is available as **@object** variable in
+                  webhook templates when a corresponding event occures.
+                  The following properties can be used to retrieve the needed information.'
+            name "#{base}"
+            refs "#{base}"
+            sections only: %w[all webhooks]
+            property :task, object_of: 'Task', desc: 'Returns the task to which this action belongs'
+            property :organization_id, Integer, desc: 'Returns the id of the organization'
+            property :organization_name, String, desc: 'Returns the name of the organization'
+            property :organization_label, String, desc: 'Returns the label of the organization'
+            unless base == Actions::Katello::Repository::Sync
+              property :content_view_id, Integer, desc: 'Returns the id of the content view'
+              property :content_view_name, String, desc: 'Returns the name of the content view'
+              property :content_view_label, String, desc: 'Returns the label of the content view'
+            end
+          end
+          jail_class = Class.new(::Actions::ObservableAction::Jail) do
+            case base
+            when Actions::Katello::Repository::Sync
+              allow :organization_id, :organization_name, :organization_label
+            else
+              allow :content_view_id, :content_view_name, :content_view_label,
+                    :organization_id, :organization_name, :organization_label
+            end
+          end
+          base.const_set('Jail', jail_class)
+        end
+      end
+    end
+  end
+end
```

Comment:
This really calls for breaking up into two concerns: `ActionsContentViewJailConcern` and `ActionsOrganizationJailConcern`.